src/index.html

```diff
@@ -37,7 +37,7 @@
 </script>
 <% } %>
 
-<% if (webpackConfig.metadata.env === 'development') { %>
+<% if (webpackConfig.metadata.ENV === 'development') { %>
```

Comment:
can we get rid of these conditions inside of template and keep template pure HTML like in https://github.com/ngParty/ngBigParty-II/blob/master/src/index.html ?